core-plugin/src/main/groovy/com/github/jrubygradle/api/gems/GemInfo.java

```diff
@@ -0,0 +1,110 @@
+package com.github.jrubygradle.api.gems;
+
+import java.net.URI;
+import java.util.List;
+
+/** GEM metadata.
+ *
+ * @author Schalk W. Cronjé
+ *
+ * @since 2.0
+ */
+public interface GemInfo {
```

Comment:
This interface could be extended, but just covered the stuff I thought necessary for now.